site/lib/queries.ts

```diff
@@ -94,6 +108,8 @@ export type PostDetails = {
   imageUrl?: string;
 };
 export type AllPosts = PostDetails[];
+export type FeaturedPost = PostDetails & { isFeaturedArticle: true };
```

Comment:
do we support featured Podcasts up there too? Might be worth supporting if it's not too hard.

Reply:
Not according to any design or concept.
I added it to the "Open UI questions" lists here https://github.com/KlimaDAO/klimadao/issues/534